### ISSUE TITLE
node.js v12 actions are deprecated and will be forced to v16 next week

### ISSUE THREAD

**Issue by mdouglass** [perforce/setup-p4#148]
https://github.blog/changelog/2023-05-04-github-actions-all-actions-will-run-on-node16-instead-of-node12/
As of May 18th, all actions will be forced to run run under node.js v16

## Expected Behavior

No warnings when the action is used in a build.

## Current Behavior

Warning when this action is used in a build:
```
Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16: perforce/setup-p4@v1. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/.
```

**Comment by aboutte** [comment]
@ryanlitalien

**Comment by abivolmv** [comment]
Would be great to have someone look at this issue ^^

**Comment by ryanlitalien** [comment]
Investigating..., thanks @abivolmv and @mdouglass.

**Comment by ryanlitalien** [comment]
Closing as fixed in #159
